planet/scripts/item_asset_types.py

```diff
@@ -78,7 +78,7 @@
     'ps4b_basic_analytic_nitf', 'ps4b_basic_analytic_rpc_nitf',
     'ps4b_basic_analytic_xml_nitf', 'ps4b_udm', 'udm', 'ortho_udm2',
     'video_file', 'video_frames', 'video_metadata', 'ortho_visual',
-    'visual', 'visual_xml'
+    'analytic', 'visual', 'visual_xml'
```

Comment:
Like "visual", "analytic" was missing from the list of default asset types. That was the core problem.